index.js

```diff
@@ -22,6 +22,10 @@ function isArray (v) {
   return Array.isArray(v);
 }
 
+function castArray (v) {
+  return isArray(v) ? v : [v];
+}
+
```

Comment:
Can you follow the pattern used by option.assets, where the files gets converted to an array only if it is a String? That way things will be more consistent